client/app/components/group/_pickupList/pickupListItem/pickupListItem.component.js

```diff
@@ -4,7 +4,7 @@ import "./pickupListItem.styl";
 
 let pickupListItemComponent = {
   bindings: {
-    data: "<",              // pickup info, including .isFull .isUserMember etc..
+    data: "<",              // pickup info, including  .isUserMember etc..
```

Comment:
I would include isFull in the data binding, to avoid using `parentCtrl` (I think that is bad style).
We could pass a `config` object that has all the settings for `pickupListItem`, e.g. isFull, isMember etc